src/Backends/Vulkan/src/factory.cpp

```
@@ -112,6 +112,8 @@ UniquePtr<IVulkanBuffer> VulkanGraphicsFactory::createBuffer(const String& name,
 		usageFlags |= VK_BUFFER_USAGE_STORAGE_BUFFER_BIT | VK_BUFFER_USAGE_INDIRECT_BUFFER_BIT;
 		alignment = m_impl->m_device.adapter().limits().minStorageBufferOffsetAlignment;
 		break;
+	default:
+		break;
```

I guess we can also throw an `InvalidArgumentException` here.